apps/designer/app/designer/features/breakpoints/use-update-canvas-width.ts

```diff
@@ -32,7 +31,16 @@ export const useUpdateCanvasWidth = () => {
     setCanvasWidth(Math.max(selectedBreakpoint.minWidth, minWidth));
   }, [selectedBreakpoint, setCanvasWidth]);
 
-  useSubscribe("canvasWidth", setCanvasWidth);
+  // Handle the case when canvas loads with a error
+  // And doesn't send any messages that it's supposed to send
+  useEffect(() => {
+    if (canvasWidth === 0) {
+      const timeout = setTimeout(() => {
+        setCanvasWidth(600);
+      }, 1000);
```

Comment:
this sounds fishy, because it's an arbitrary number, what if it loads with an error and it takes longer?

Reply:
The delay is to avoid setting it to 600 before it's set to a breakpoint value, which causes a glitch. If it takes longer to load the error that's not an issue.